frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -120,17 +119,26 @@ const RealPhoneForm = (props: RealPhoneFormProps) => {
       <form onSubmit={onSubmit} className={styles.form}>
         {renderErrorMessage}
 
-        <input
-          className={`${styles["c-verify-phone-input"]}`}
+        <PhoneInput
+          className={`${styles["PhoneInput"]} ${
+            phoneNumberError ? "is-error" : ""
+          }`}
           placeholder={l10n.getString(
             "phone-onboarding-step2-input-placeholder"
           )}
-          ref={phoneNumberField}
+          countries={["US", "CA"]}
+          flags={flags}
+          defaultCountry="US"
           type="tel"
+          international
+          autoComplete="tel"
+          limitMaxLength
+          value={phoneNumber}
           required={true}
-          onChange={onChange}
           autoFocus={true}
+          onChange={(number: E164Number) => setPhoneNumber(number ?? "")}
```

Comment:
Doesn't this _always_ get a string?

```suggestion
          onChange={(number: E164Number) => setPhoneNumber(number)}
```

Reply:
Weird - it was complaining before about it being potentially undefined - updated.